### PR TITLE
Drupal Security Patch

### PR DESCRIPTION
**Pull request by alexfornuto**
## Effect
PR includes the following changes:
- Prep work for Drupal Security patch to be released today.

## Remaining Work
- [x] Flesh out doc
- [x] Apply Actual Patch
- [ ] Tech Review
- [ ] Copy Review

### CODE REVIEW

File: source/_docs/drupal-security-patches.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+</div>
+
+</div>
+
+## Lock Multidev Environments
+
+As an additional security measures, sites with [Multidev](/docs/multidev/) environments should consider [locking](/docs/security/) them until they can be patched. If you have [Terminus](/docs/terminus/) installed on your local computer, you can lock all environments at once with the following Bash script:
+
+```bash
```
Comment (alexfornuto) [inline source/_docs/drupal-security-patches.md]: @greg-1-anderson I humbly submit:

Comment (greg-1-anderson) [reply source/_docs/drupal-security-patches.md]: Patch LGTM

File: source/_docs/drupal-security-patches.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    ```bash
+    curl -O https://patch-diff.githubusercontent.com/raw/pantheon-systems/drops-8/pull/206.patch
+    patch -p1 < 206.patch
+    ```
+
+4. Remove the patch file, commit and push your changes back to Pantheon:
+
+    ```bash
+    rm 206.patch
+    git commit -m "Mannually apply Drupal security patch"
```
Comment (ari-gold) [inline source/_docs/drupal-security-patches.md]: typo: Manually

File: source/_docs/drupal-security-patches.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    ```bash
+    curl -O https://patch-diff.githubusercontent.com/raw/pantheon-systems/drops-7/pull/119.patch
+    patch -p1 < 119.patch
+    ```
+
+4. Remove the patch file, commit and push your changes back to Pantheon:
+
+    ```bash
+    rm 119.patch
+    git commit -m "Mannually apply Drupal security patch"
```
Comment (ari-gold) [inline source/_docs/drupal-security-patches.md]: "Manually apply patch for Drupal 7.58"

File: source/_docs/drupal-security-patches.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+    ```bash
+    curl -O https://patch-diff.githubusercontent.com/raw/pantheon-systems/drops-8/pull/206.patch
+    patch -p1 < 206.patch
+    ```
+
+4. Remove the patch file, commit and push your changes back to Pantheon:
+
+    ```bash
+    rm 206.patch
+    git commit -m "Manually apply Drupal security patch"
```
Comment (ari-gold) [inline source/_docs/drupal-security-patches.md]: git commit -am

**Pull request merged**